polars/query.py

```diff
@@ -318,15 +318,13 @@
         .group_by("CounterID")  # GROUP BY CounterID
         .agg(
             [
-                pl.col("URL")
-                .map_elements(lambda y: len(y), return_dtype=pl.Int64)
```

Comment:
This was wrong. Didn't compute the `AVG` and it uses a python function, where we have native expressions for this.